data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,472 @@ mission "Unfettered returning home"
+			# This is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles or so.
+			`	He suddenly looks at you intensively, straight in the eyes. "We sometimes bend the rules for our friends, but that is not the case with you," he says with a sudden change of tone before leaving.`
+				to display
+					"reputation: Hai (Unfettered)" < -2000
+			`	Mera comes back, looking a bit impatient. "Are you going to talk all day, or are you going to fight?" The Unfettered pilots immediately head to their ships. The engineer pulls you in close once they are far enough away. "I want this done by the end of the day, so do not try my patience. You are lucky I offered this deal in the first place, as they were planning to take the relic by force. Now, quick, get to your ship!" You see the Unfettered have already begun their launching procedures, so you prepare for takeoff as soon as you can.`
+			`	(Remember to park your other ships and make sure your flagship is the Anomalocaris.)`
+				accept
+			label headingout
+			action
+				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+				set "unfettered: fled with the relic"
+			`	Seeing as they are unlikely to cooperate, you quickly head to your ship and launch before they have time to react, but shortly after they begin to follow you.`
+				launch
+	on enter
+		conversation
+			branch fleeing
+				has "unfettered: fled with the relic"
+			branch "too many ships"
+				"total ships" > 1
+			`As you leave the planet's atmosphere, you see the challengers already waiting for you in the system. They hail you, "Luck has nothing to do with it. Show us what you got, monkey! Show us that you control your weapons well, as any warriors should. Prove to us that you have the strength to disable our ship, yet still have the restraint not to destroy them."`
+				accept
+			label fleeing
+			action
+				fail
+			`The Unfettered don't seem too pleased to see you leaving with their prize. The challengers are already waiting for you in orbit, and hail you. "Where do you think you're going, monkey?"`
+				decline
+			label "too many ships"
+			`As you take to join your fleet in orbit, You receive a hail from the Unfettered, "If you can't even count how many ships you have you're not worthy of our time, monkey!"`
+			action
+				fail
+	on disabled
+		fail
+		conversation
+			`The enemy ships hail you. "Thought you could get away from us so easily? You better bribe us and then land on our worlds quick, before we value blowing you up more than preserving that ship!"`
+				to display
+					has "unfettered: fled with the relic"
+				decline
+			`The enemy ships hail you. "You have failed the challenge. We will take the ship from you when we see fit. I'll fix your ship and leave you alone for a bribe, but I suggest you land on an Unfettered world and return the ship to us, with a most generous compensation. If you try to keep it from us, we'll do things the hard way."`
```

Comment:
I vs. we?

Reply:
This sounds fine. The "we" is referring to them as a group taking the ship away, and the "I" is the Hai that is speaking, about suggesting how to deal with their people. Any we or us refers to the Uhai, and the I's refer to the individual speaking.

Reply:
But it is "ships" hailing you, so I is out of place.

Reply:
It's still the "I" as in the pilot of the ship who hailed you, and them saying "we' as in referring to them group taking the ship, and "us" being the Uhai. The pilot is using "I" as in they themselves will repair your ship for a bribe (not the entire group flying at you and repairing your ship", and "I" suggest landing, as they're the one sending the hail, not all the other ships.

Reply:
I'm not sure I get what you're saying. It explicitly says "ships," so I would assume that they are all hailing.

Reply:
Yeah. So I'd say the error is in "The enemy ships hail you", and it should be, "One of the ships hail you", because the whole conversation comes from one specific pilot/is all in one quote block"; rather than change several parts of the dialog, the plurality should be changed. Less changes = better.

Reply:
> I'm not sure I get what you're saying. It explicitly says "ships," so I would assume that they are all hailing.

It seems to be only one ship talking. Also, perhaps it should be changed to this:

```suggestion
			`The one of the enemy ships hails you. "You have failed the challenge. We will take the ship from you when we see fit. I'll fix your ship and leave you alone for a bribe, but I suggest you land on an Unfettered world and return the ship to us, with a most generous compensation. If you try to keep it from us, we'll do things the hard way."`
```

Reply:
Forgot to take out the The 😉.
```suggestion
			`One of the enemy ships hails you. "You have failed the challenge. We will take the ship from you when we see fit. I'll fix your ship and leave you alone for a bribe, but I suggest you land on an Unfettered world and return the ship to us, with a most generous compensation. If you try to keep it from us, we'll do things the hard way."`
```